js/src/pages/ProjectTeamPage.tsx

```diff
@@ -40,17 +102,72 @@ export const ProjectTeamPage: React.FC = () => {
     fetchProject(slug);
   }, [slug, fetchProject]);
 
+  useEffect(() => {
+    fetchEmployees();
+  }, [fetchEmployees]);
+
   if (!project || project.slug !== slug) return null;
 
   if (!team || team.id !== id) return null;
 
   return (
     <Row gutter={10} className="content">
-      <Col lg={12} md={24} xs={24}>
-        <Card title="Team" extra={<PlusOutlined />}>
-          <TeamMembersList members={team.members} roles={project.roles} />
+      <Col xl={14} lg={24} md={24} xs={24}>
+        <Card
+          title={team.name}
+          extra={<PlusOutlined onClick={openAddMemberForm} />}
+        >
+          <List
+            itemLayout="horizontal"
+            dataSource={team.members}
+            renderItem={(item) => (
+              <TeamMembersListItem
+                roles={project.roles}
+                member={item}
+                edit={editMemberRole}
+                teamId={team.id}
+                remove={removeMember}
+              />
+            )}
+          />
+        </Card>
+
+        <Card>
+          {team.status === "Active" ? (
+            <TeamChangeStatus
+              description="Access to tasks is closed. Can be restored when needed."
```

Comment:
Very strange description. Looks like **now** access to tasks is closed. There are you should say what will happen if you press this button.

Reply:
Fixed